docs/advanced_topics/security_considerations.md

```diff
@@ -0,0 +1,111 @@
+# Security considerations
+
+Wagtail is built on Django, and supports all Django 's security features. However, being a CMS, there are some beneficial features that might not ordinarily come to mind when using Django as is, but should be given some thought. For more context, here's Django's [documentation on security.](inv:django#topics/security)
+
+```{contents}
+---
+local:
+depth: 1
+---
+```
+
```

Comment:
Should we add another section here above the CSP one with additional Wagtail security references.

* https://docs.wagtail.org/en/stable/topics/images.html#security-considerations
* https://docs.wagtail.org/en/stable/advanced_topics/documents/storing_and_serving.html#security-considerations
* https://docs.wagtail.org/en/stable/deployment/under_the_hood.html#security (Deployment security)
* Reporting security issues https://docs.wagtail.org/en/stable/contributing/security.html